editor/script_editor_debugger.cpp

```diff
@@ -1686,6 +1686,45 @@ void ScriptEditorDebugger::_clear_remote_objects() {
 	remote_objects.clear();
 }
 
+void ScriptEditorDebugger::_clear_errors_list() {
+
+	error_list->clear();
+	error_count = 0;
+	_notification(NOTIFICATION_PROCESS);
+}
+
+// Right click on specific file(s) or folder(s).
+void ScriptEditorDebugger::_error_list_item_rmb_selected(int p_item, const Vector2 &p_pos) {
+
+	item_menu->clear();
+	item_menu->set_size(Size2(1, 1));
+
+	// Allow specific actions only on one item.
+	bool single_item_selected = error_list->get_selected_items().size() == 1;
+
+	if (single_item_selected) {
+		item_menu->add_icon_item(get_icon("CopyNodePath", "EditorIcons"), TTR("Copy Error"), ITEM_MENU_COPY_ERROR);
+	}
+
+	if (item_menu->get_item_count() > 0) {
+		item_menu->set_position(error_list->get_global_position() + p_pos);
+		item_menu->popup();
+	}
+}
+
+void ScriptEditorDebugger::_item_menu_id_pressed(int p_option) {
+
+	switch (p_option) {
+
+		case ITEM_MENU_COPY_ERROR: {
+			String title = error_list->get_item_text(error_list->get_current());
+			String desc = error_list->get_item_tooltip(error_list->get_current());
+
+			OS::get_singleton()->set_clipboard(title + "\n----------\n" + desc);
```

Comment:
I don't personally like things getting copied to my clipboard that weren't on the screen. All the extra newlines also makes copy/pasting into say, IRC really cumbersome. I'd much rather this be
"title + ": " + desc"

Reply:
Errors in "Debugger->Errors" tab are mostly useless without extended description that is placed in error's tooltip.

Reply:
As the author explained on IRC the 'desc' field itself actually has multiple lines in it already so maybe this change doesn't make much sense. It may be nice to put the desc somewhere in the error list though. But maybe that doesn't make sense either.

If most users don't think that the clipboard function needs any discussion I'll happily retract my objection.